### PR TITLE
Build man page

### PR DESCRIPTION
**Pull request by izgzhen**
It addresses #246

### CODE REVIEW

**Comment by izgzhen** [comment]
The major concern is `buildUsersGuideHtml`: I found the current implementation of `Sphinx` builder can't quite elegantly map to the `sphinx` function as in `rules/sphinx.mk`, so I need to use `cmd`.

**Comment by snowleopard** [comment]
@izgzhen Wait, have you seen this: #413? Aren't you duplicating the functionality?

**Comment by izgzhen** [comment]
> Wait, have you seen this: #413? Aren't you duplicating the functionality?

I thought that PR hasn't implemented the users_guide and man page yet, does it?

**Comment by bgamari** [comment]
> I thought that PR hasn't implemented the users_guide and man page yet, does it?

I believe it does. For instance, grep for `man` in the diff.

**Comment by snowleopard** [comment]
@izgzhen It definitely builds the User's Guide. As far as I understand, documentation is mostly done, we only miss a few bits listed here: #414. 

I've now closed #246 to avoid confusion.

File: src/Settings/Builders/Sphinx.hs [hunk 1]
```diff
@@ -16,7 +16,7 @@ sphinxBuilderArgs = do
                 , arg =<< getInput
                 , arg outPath ]
             , builder (Sphinx Man) ? mconcat
-                [ arg "-b", arg "latex"
+                [ arg "-b", arg "man"
```
Comment (snowleopard) [inline src/Settings/Builders/Sphinx.hs]: This does look like the right change though.

@patrickdoc Was this a typo?

Comment (patrickdoc) [reply src/Settings/Builders/Sphinx.hs]: Ah, yes, my apologies. That should indeed be `man` not `latex`.

**Comment by izgzhen** [comment]
Ah, I see. I've confirmed that `users_guide` is indeed built -- but it still looks that man page is missing.

**Comment by snowleopard** [comment]
> but it still looks that man page is missing

@izgzhen Thanks, I've added this into the todo list in #414. Hopefully, we can fix it in this PR!

**Comment by izgzhen** [comment]
revised

**Comment by snowleopard** [comment]
@izgzhen Thanks! Looks good. Broken CI is due to #426.

**Pull request merged**